pkg/cninetwork/weave_mac.go

```diff
@@ -0,0 +1,10 @@
+// +build darwin
+
+package cninetwork
+
+import "github.com/vishvananda/netlink"
+
+func linkToNetDev(link netlink.Link) (Dev, error) {
+
+	return Dev{}, nil
```

Comment:
cool, does this mean we can build and lint this on Mac now?

Reply:
You can run a selection of the tests now, including the ones I added today. I've also updated the compose parser so that we can "fake" the input OS.